context/app/static/js/components/home-revision/AssayTypeBarChart/utils.js

```diff
@@ -0,0 +1,35 @@
+import produce from 'immer';
+/* eslint-disable no-param-reassign */
```

Comment:
Is this only for the sort call below?

Reply:
This is for the immer portion which includes all of the reassignments of the draft param.

Reply:
Ah, right, ok, didn't catch that one.